File: 3581-the-two-sneaky-numbers-of-digitville/the-two-sneaky-numbers-of-digitville.java
class Solution {
    public int[] getSneakyNumbers(int[] nums) {
        Map<Integer,Integer> map = new HashMap<>();
        for(int i:nums){
            map.put(i,map.getOrDefault(i,0)+1);
        }
        int[] arr = new int[2];
        int ind = 0;
        for(int i:map.keySet()){
            if(map.get(i)==2){
                arr[ind++] = i;
            }
        }
        return arr;
    }
}

File: 3581-the-two-sneaky-numbers-of-digitville/the-two-sneaky-numbers-of-digitville.cpp
class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& nums) {
        vector<int> list;
        set<int> set;
        for(int i:nums){
            if(set.count(i)>0){
                list.push_back(i);
            }
            set.insert(i);
        }
        return list;
    }
};